src/ui/public/agg_response/tabify/_buckets.js

```diff
@@ -81,4 +84,23 @@ TabifyBuckets.prototype._orderBucketsAccordingToParams = function (params) {
   }
 };
 
+// dropPartials should only be called if the aggParam setting is enabled,
+// and the agg field is the same as the Time Range.
+TabifyBuckets.prototype._dropPartials = function (params, timeRange) {
+  if (!timeRange ||
+    this.buckets.length <= 1 ||
+    this.objectMode ||
+    params.field.name !== timeRange.name) return;
+
+  const interval = this.buckets[1].key - this.buckets[0].key;
+
+  this.buckets = this.buckets.filter(bucket => {
+    if (bucket.key < timeRange.gte) return false;
+    if (bucket.key + interval > timeRange.lte) return false;
```

Comment:
nit: it's better to align with our styleguide [always-use-braces-for-conditionals-and-loops](https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#always-use-braces-for-conditionals-and-loops)